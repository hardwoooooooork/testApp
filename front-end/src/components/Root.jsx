import React, { Component } from 'react'
import { HashRouter } from 'react-router-dom';
import Layout from './layouts'

export default class Root extends Component {
    render() {
        return (
            <HashRouter>
                <Layout
                    {...this.props}
                />
            </HashRouter>
        )
    }
}
