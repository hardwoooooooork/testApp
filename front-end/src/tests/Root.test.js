import React from 'react';
import { render } from '@testing-library/react';
import Root from '../components/Root';
describe('<Root />', () => {
    it('matches snapshot', () => {
        const utils = render(<Root />);
        expect(utils.container).toMatchSnapshot();
    });
});