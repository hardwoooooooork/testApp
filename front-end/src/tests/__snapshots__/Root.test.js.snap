// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Root /> matches snapshot 1`] = `
<div>
  <div
    class="ui center aligned middle aligned grid"
    style="height: 100vh;"
  >
    <div
      class="column"
      style="max-width: 450px;"
    >
      <h2
        class="ui teal center aligned header"
      >
        <img
          class="ui image"
          src="kakao.png"
        />
         Log-in to your account
      </h2>
      <form
        class="ui large form"
      >
        <div
          class="ui stacked segment"
        >
          <div
            class="required field"
          >
            <div
              class="ui fluid left icon input"
            >
              <input
                name="username"
                placeholder="ID"
                required=""
                type="text"
                value=""
              />
              <i
                aria-hidden="true"
                class="user icon"
              />
            </div>
          </div>
          <div
            class="required field"
          >
            <div
              class="ui fluid left icon input"
            >
              <input
                name="password"
                placeholder="Password"
                required=""
                type="password"
                value=""
              />
              <i
                aria-hidden="true"
                class="lock icon"
              />
            </div>
          </div>
          <button
            class="ui teal large fluid button"
          >
            Login
          </button>
        </div>
      </form>
      <div
        class="ui message"
      >
        New to us? 
        <span
          style="color: blue; margin-right: 7px; cursor: pointer;"
        >
          Sign Up
        </span>
      </div>
    </div>
  </div>
</div>
`;
